android/src/main/kotlin/com/example/android_plugin/AndroidPlugin.kt

```diff
@@ -96,18 +164,27 @@ class AndroidPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
 
     override fun onDetachedFromActivity() {
         Log.i("onDetachedFromActivity", "Function not implemented")
-
     }
 
     override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?): Boolean {
         Log.i("onActivityResult", "Function not implemented")
-        return false
+        return true
     }
 
     companion object AndroidPlugin {
         const val REQUEST_CODE = 500
         const val PERMISSION_CHECK_METHOD = "permissionCheck"
         const val PERMISSION_REQUEST_METHOD = "requestPermission"
         const val CHANNEL = "android_plugin"
+        const val GET_LOCATION = "getLocation"
+        const val INITIALIZE = "initialize"
+        const val STOP_LOCATION_REQUEST = "stopLocationRequest"
+        const val INTERVAL: Long = 5000
+        const val LOCATION_STOPPED = "Location request stopped"
+        const val ERROR = "Error"
+        const val PERMISSION_DENIED = "Permission denied"
+        const val LOCATION_CHANNEL = "location_channel"
+        const val LONGITUDE_TAG = "Longitude:"
```

Comment:
I think you missed a white space here after ':'

Reply:
Fixed